src/pages/home/report/ReportActionCompose.js

```diff
@@ -734,6 +847,29 @@ class ReportActionCompose extends React.Component {
                     <ExceededCommentLength commentLength={encodedCommentLength} />
                 </View>
                 {this.state.isDraggingOver && <ReportDropUI />}
+                {!_.isEmpty(this.state.suggestedEmojis) && this.state.shouldShowSuggestionMenu && (
+                    <ArrowKeyFocusManager
```

Comment:
I am wondering why didn't we add the `ArrowKeyFocusManager` as a part of the `EmojiSuggestions` itself?

Reply:
@perunt did you get a chance to look at this comment ?

Reply:
I decided not to split the keyboard processing logic into two separate files on the same screen because it could lead to confusion in the future. Keeping the logic in a single file helps to maintain clarity and reduces the chances of introducing errors. What do you think?